embedding/cpp/elm_flutter_view.cc

```diff
@@ -51,12 +51,7 @@ void ElmFlutterView::SetEngine(std::unique_ptr<FlutterEngine> engine) {
 void ElmFlutterView::Resize(int32_t width, int32_t height) {
   assert(IsRunning());
 
-  int32_t view_width = width, view_height = height;
-  evas_object_geometry_get(evas_object_, nullptr, nullptr, &view_width,
-                           &view_height);
-  if (view_width != width || view_height != height) {
-    FlutterDesktopViewResize(view_, width, height);
-  }
+  evas_object_resize(evas_object_, width, height);
```

Comment:
I think this is not appropriate because it depends on resize event callback of view_.
I think this Resize method is an API for view_. I don't think we need to manipulate evas_object_ directly in this Resize method.

Reply:
All right, your opinion makes sense, and this PR doesn't make any functional changes, so I'll close this PR.